platform/android/export/export_plugin.cpp

```diff
@@ -2417,6 +2421,10 @@ bool EditorExportPlatformAndroid::has_valid_export_configuration(const Ref<Edito
 			err += template_err;
 		}
 	} else {
+#ifdef ANDROID_ENABLED
+		err += TTR("Gradle build is not supported for the Android editor");
```

Comment:
I believe we always add a newline so we can chain those errors, no?
```suggestion
		err += TTR("Gradle build is not supported for the Android editor") + "\n";
```